kdwsdl2cpp/src/converter_complextype.cpp

```diff
@@ -389,6 +389,20 @@ QString Converter::generateMemberVariable(const QString &rawName, const QString
     newClass.addFunction(setter);
     newClass.addFunction(getter);
 
+    if (optional) {
+        KODE::Code checkerCode;
+        if (usePointer) {
+            checkerCode += QLatin1String("return ") + variableName + QLatin1String(".isNull() == false;");
```

Comment:
I would have made it

    return !foo.isNull();

and

    return !foo_nil;

rather than comparing to false, as a more common way of writing this (at least in the Qt world).